src/components/Composer/index.android.tsx

```diff
@@ -1,100 +1,94 @@
-import PropTypes from 'prop-types';
-import React, {useCallback, useEffect, useMemo, useRef} from 'react';
-import {StyleSheet} from 'react-native';
-import _ from 'underscore';
+import React, {ForwardedRef, useCallback, useEffect, useMemo, useRef} from 'react';
+import {StyleProp, StyleSheet, TextInput, TextStyle} from 'react-native';
 import RNTextInput from '@components/RNTextInput';
 import * as ComposerUtils from '@libs/ComposerUtils';
 import themeColors from '@styles/themes/default';
 
-const propTypes = {
+type ComposerProps = {
```

Comment:
Create `types.ts` file and put `ComposerProps` there. Make sure you marked all necessary params as optional.

Reply:
Done!